### PR TITLE
Tganley/bugfix/iris fs test plan fixes

### PR DESCRIPTION
**Pull request by tganley**

### CODE REVIEW

**Comment by tganley** [comment]
This was originally meant to be merged into Jenish's Iris branch. It's all tested on the flatsat.

File: ex2_system/source/diagnostic/diagnostic.c [hunk 1]
```diff
@@ -320,69 +321,85 @@ static void adcs_watchdog_daemon(void *pvParameters) {
...
             eps_set_pwr_chnl(PYLD_5V0_PWR_CHNL, OFF);
 
             // Allow the system to fully power off
             vTaskDelay(reset_wait_period);
 
-            // Check that the NS payload has been turned off.
+            // Check that the payload has been turned off.
             if ((eps_get_pwr_chnl(PYLD_3V3_PWR_CHNL) != OFF) || (eps_get_pwr_chnl(PYLD_5V0_PWR_CHNL) != OFF)) {
-                ex2_log("NS payload failed to power off.");
+                ex2_log("Payload failed to power off.");
```
Comment (Taylorr82) [inline ex2_system/source/diagnostic/diagnostic.c:380]: sys_log

**Review by Taylorr82 — CHANGES_REQUESTED** [review]

File: ex2_hal/iris/hardware_interface/source/iris.c [hunk 2]
```diff
@@ -174,7 +174,7 @@ Iris_HAL_return iris_transfer_image(uint32_t image_length) {
     fptr = red_open("iris_image.jpg", RED_O_CREAT | RED_O_WRONLY);
 
     if (fptr == NULL) {
```
Comment (Taylorr82) [inline ex2_hal/iris/hardware_interface/source/iris.c:176]: Doesn't this return -1 on failure?

Comment (tganley) [reply ex2_hal/iris/hardware_interface/source/iris.c:176]: Yes! Bad Jenish

**Review by Taylorr82 — APPROVED** [review]

**Pull request merged**